<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Login </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="lineEdit_Password_login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_Password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>19</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Email_login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_Email">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>19</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_login">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: blue;
font-size: larger;
font-weight: 400;
border: none;
border-radius: 20px;
background-color: white;</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_SignUp_instead">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: blue;
font-size: larger;
font-weight: 400;
border: none;
border-radius: 20px;
background-color: white;</string>
     </property>
     <property name="text">
      <string>Sign Up
 Instead</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Seller">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Seller</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Password_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>19</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>I am a:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Admin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_User">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
